lib/Genesis/Kit.pm

```diff
@@ -242,6 +253,13 @@ sub metadata {
 	return $self->{__metadata};
 }
 
+# }}}
+# uses_credhub - does this kit use credhub instead of vault {{{
+sub uses_credhub {
+	my ($self) = @_;
+	return defined($self->metadata->{secret_store}) && $self->metadata->{uses_credhub} eq "credhub";
```

Comment:
second value should also be "uses_credhub"

Reply:
fixed